pipeline/update_watchdog.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+# Assumptions:
+# experiment-deploy & nodedb repos are set up in the path
+# python3 installed & requests package is installed
+# github credentials for nodedb exist for current user
+# ssh watchdog is correctly configured
+
+base=`pwd`
```

Comment:
a better practice: https://stackoverflow.com/questions/192319/how-do-i-know-the-script-file-name-in-a-bash-script